mbf_abstract_nav/include/mbf_abstract_nav/abstract_execution_base.h

```diff
@@ -45,61 +45,75 @@
 
 #include <mbf_abstract_nav/MoveBaseFlexConfig.h>
 
+#include <string>
+
 namespace mbf_abstract_nav
 {
-
+/**
+ * @brief Base class for running concurrent navigation tasks.
+ *
+ * The class uses a dedicated thread to run potentially long-lasting jobs.
+ * The user can use waitForStateUpdate to get notification about the progress
+ * of the said job.
+ */
 class AbstractExecutionBase
 {
  public:
+   AbstractExecutionBase(const std::string& name);
 
-  AbstractExecutionBase(std::string name);
+   virtual ~AbstractExecutionBase();
 
-  virtual bool start();
```

Comment:
The indentation is 2 spaces, so it was correct

Reply:
Yep, that definitely odd, but I fear this is an artefact of the "partial" formatting we apply. If i edit it manually, our CI will fail. The only thing I can think of is to reformat the whole file - and im not sure if its really worth it

Reply:
No idea;  anyway, I formatted locally

Reply:
lgtm